logger_test.go

```diff
@@ -198,6 +202,15 @@ func TestLoggerFields(t *testing.T) {
 	t.Run("test that fields are populated correctly", func(t *testing.T) {
 		expectedHostname, _ := os.Hostname()
 		os.Setenv("RELEASE", "mycoolrelease")
+
+		var ddtags string
+		if _, err := os.Stat("/.dockerenv"); err == nil {
+			containerId, _ := getContainerId()
+			ddtags = fmt.Sprintf("container_id:%s", containerId)
+		} else {
+			ddtags = ""
+		}
```

Comment:
strings are `""` by default, so redundant
```suggestion
		}
```